tests/test_device.py

```diff
@@ -378,6 +378,63 @@ def test_sample_attribute_error(self, mock_device_with_paulis_and_methods):
             mock_device_with_paulis_and_methods.execute(queue, observables)
 
 
+class TestParameters:
+    """Test for checking device parameter mappings"""
+
+    @pytest.fixture
```

Comment:
I like it if the scope is explicit:
```suggestion
    @pytest.fixture(scope="function")
```